plugin-packs/postcss-preset-env/src/index.js

```diff
@@ -9,23 +9,24 @@ import writeToExports from './lib/write-to-exports';
 import getOptionsForBrowsersByFeature from './lib/get-options-for-browsers-by-feature';
 import { pluginIdHelp } from './lib/plugin-id-help';
 import { pluginHasSideEffects } from './lib/plugins-with-side-effects';
+import { log, dumpLogs } from './lib/log-helper';
 
 const DEFAULT_STAGE = 2;
 const OUT_OF_RANGE_STAGE = 5;
 
-const logDebug = msg => console.log('[postcss-preset-env] -> %s', msg);
-
 const plugin = opts => {
 	// initialize options
 	const options = Object(opts);
 	const features = Object(options.features);
-	const debug = options.debug ? logDebug : () => {};
 	const featureNamesInOptions = Object.keys(features);
 	const insertBefore = Object(options.insertBefore);
 	const insertAfter = Object(options.insertAfter);
 	const browsers = options.browsers;
 	let stage = DEFAULT_STAGE;
 
+	// Forcing logs to be emptied between runs
+	dumpLogs();
```

Comment:
maybe a `resetLogger` function?
That would not require a code comment :)

Reply:
You're more than right!